README.md

```diff
@@ -0,0 +1,208 @@
+
+## A. Preparation
+
+## 1. Kobiton username and API key
+
+In order to execute automation test on Kobiton devices, you need to prepare your Kobiton username and API key for authenticating with Kobiton. 
+  
+> Please visit https://portal.kobiton.com/register to create a new Kobiton account if you do not have one.
+
+Follow instructions at `IV. Configure Test Scripts for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) to get Kobiton username and API key.
+
+## 2. Sample
+
+We have provided a sample for automation test with Travis CI configuration file in this repository.
+
+The sample includes:
+- Basic configuration for Travis CI (`.travis.yml`)
+- Simple automation test script written in NodeJs. (`samples/automation-script` folder)
+
+> After this guideline, you can make your own adjustment to meet your requirements.
+
+To use the provided sample, follow these steps:
+
+1. Fork this repository https://github.com/kobiton/Travis-HockeyApp-Appium-ReactNative/tree/master
+2. Synchronize the forked repository with Travis CI. Visit [Travis CI document](https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/#how-to-migrate-a-private-repository-to-github-apps) for detailed instruction.
+3. Clone the forked repository.
+
+The initial build on Travis CI will fail as we haven't done any configuration. Please follow instructions in the next parts to configure for automation testing.
+
+## B. Setup
+
+This part will guide you how to set your Kobiton username and API key as environment variables and apply Kobiton device's desired capabilities to your automation test script. After that, we will configure Travis CI to execute the test on your preferred platform (Android/ iOS)
+
+## 1. Setting environment variables
+
+  Go to your Travis CI build project settings.
+
+  Add these two environment variable:
+  
+  -  KOBITON_USERNAME : Your Kobiton's username.
+  -  KOBITON_API_KEY : Your Kobiton's API Key.
+  
+  Please make sure to hide `KOBITON_API_KEY` value so it won't be exposed in build log.
+  
+  The result should look like below:
+  
+  ![](docs/assets/travis_env.jpg)
+
+
+## 2. Getting desired capabilities
+
+  The desired capabilities need to be added to the automation test script in order for tests to be executed on the Kobiton device.
+
+  Go to https://portal.kobiton.com and login to your Kobiton account.
+  1. In the top navigation bar, select **"Devices"**.
+  
+  ![](docs/assets/1_device_bar.jpg)
+
+  2. Pick an Android or iOs device if you want to execute tests on Android/ iOS apps. Hover over the chosen device and click on the Automation settings button (the gear symbol).
+  
+  ![](docs/assets/2_get_device.jpg)
+
+  1. In the popup window, you can choose the language of your test scripts and application type. The desired capabilities code will be generated in the right panel.
+
+  In the provided sample scripts, we will be testing:
```

Comment:
Totally, lost in this part

Reply:
What is the point of it ?

Reply:
After a while scrolling up and down, reading through nearly all parts in the document, finally I can understand it. 

I think this is a problem, and endusers will face with the same issue like me. Think about it